src/Pages/Shop/ShopDetail/DetailContent/LeftSection/LeftSection.js

```diff
@@ -0,0 +1,148 @@
+        for Photoshop, from rough and gritty to smooth and delicate and
+        everything between.
+      </DescriptionSubTitle>
+      <DescriptionMainText>
+        Each brush is built from scratch using original analog brush and texture
+        samples to create a diverse and powerful set of tools that combine the
+        tactile beauty of ink on paper with the convenience of a modern digital
+        workflow.
+      </DescriptionMainText>
+      <DescriptionMainText>
+        With a huge range of authentic stroke styles, textures, edge effects,
+        poses, and brush feels The Rusty Nib Brush Set is as realistic as it is
+        diverse. We've got you covered whether you're re-creating the gritty
+        aesthetic of scanned print ephemera or aiming for the subtlety of real
+        india ink flowing onto the page.
+      </DescriptionMainText>
+      <DescriptionTitle>
+        <br /> &nbsp;
+      </DescriptionTitle>
+      <Video />
+      <GetItemInfo>Here's what you get:</GetItemInfo>
+      <ListItem>
+        {text.text.data &&
+          text.text.data.map((v) => {
+            return <ListItemDescription>{v}</ListItemDescription>;
+          })}
+      </ListItem>
+      <ContainerP>&nbsp;</ContainerP>
+    </Container>
+  );
+}
+
+export default LeftSection;
+
+const Container = styled.section`
+  width: 60%;
+  height: 3421.94px;
+  margin-right: 30px;
+  padding-left: 30px;
+  padding-right: 30px;
+  background-color: green;
+`;
+
+const TitleContent = styled.div`
+  display: flex;
+  align-items: center;
+  height: 173px;
+  width: 100%;
+  background-color: purple;
+`;
+
+const TitleContentBox = styled.div`
+  height: 102px;
+  background-color: red;
+`;
+
+const Title = styled.h1`
+  height: 33px;
+  margin-bottom: 13px;
+  font-size: 27.64px;
+`;
+
+const Sub = styled.div`
+  font-size: 18px;
+  letter-spacing: 0.8px;
+  line-height: 28px;
+`;
+
+const DescriptionTitle = styled.h4`
+  margin-top: 30px;
+  margin-bottom: 15px;
+  font-size: 18px;
+  line-height: 24px;
+`;
+
+const DescriptionSubTitle = styled.h2`
+  height: 150px;
+  line-height: 30px;
+  margin-top: 30px;
+  margin-bottom: 15px;
+`;
+
+const DescriptionMainText = styled.p`
+  font-size: 17px;
+  line-height: 29px;
+  letter-spacing: 0px;
+  margin-bottom: 22px;
+`;
+
+const Video = styled.iframe.attrs((props) => ({
+  src:
+    props.video ||
+    "https://player.vimeo.com/video/352387513?color=bf914c&title=0&byline=0&portrait=0",
+}))`
+  width: 100%;
+  height: 453.16px;
+`;
+
+const GetItemInfo = styled.h3`
+  height: 56px;
+  line-height: 84px;
+  margin-top: 30px;
+  margin-bottom: 15px;
+`;
+
+const ListItem = styled.ul`
+  margin-left: 25px;
+  margin-bottom: 12.5px;
+`;
+
+const ListItemDescription = styled.li({
+  paddingLeft: "15px",
+  marginBottom: "4px",
+  background: "red",
+  "::before": {
+    content: "'✓'",
+    position: "relative",
+    left: "-13%",
+  },
+});
```

Comment:
```suggestion
const ListItemDescription = styled.li`
  padding-left: 15px;
  margin-bottom: 4px;
  background: red;
  &::before {
    content: "✓";
    position: relative;
    left: -13%;
  }
`;
```
정상적인 css 문법으로 작성해주세요

Reply:
수정했습니다!

Reply:
수정하였습니다!